analysis_options.yaml

```diff
@@ -0,0 +1,83 @@
+include: package:pedantic/analysis_options.yaml
```

Comment:
Do you want to include all the lints below (or just use the ones specified by `pedantic`?)?

Reply:
I like the ones below, generally. The ones that are commented out and "todos"